web/client-api/src/main/java/io/deephaven/web/client/api/ServerConfigValues.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.web.client.api;
+
+/**
+ * A container for configuration values from the server.
+ */
+public interface ServerConfigValues {
+    String GRADLE_VERSION = "gradleVersion";
+    String VCS_VERSION = "vcsVersion";
+    String JAVA_VERSION = "javaVersion";
```

Comment:
These constants aren't used on Community at all, so doesn't make sense here.

Reply:
In DHC, we have a convention for versions to be read at runtime. By default in dhc this includes:
 * `java.version`
 * `deephaven.version`
 * `barrage.version`